pkg/config/configobserver/configobserver_test.go

```diff
@@ -98,7 +98,8 @@ func TestChangeSupportConfig(t *testing.T) {
 
 }
 
-func TestChangeObserved(t *testing.T) {
+// ignore until resolved
```

Comment:
Are we tracking this somewhere? This comment don't give me and idea, we should probably add a BZ id here

Reply:
I added reference to this PR in the internal case for investigation

Reply:
ok